packages/push-macros/src/push_state/printing/generate_builder.rs

```diff
@@ -163,7 +163,7 @@ pub fn generate_builder(
                         ///
                         /// let mut state = PushState::builder()
                         ///     .with_max_stack_size(100)
-                        ///     .with_program([])?
+                        ///     .with_no_program()
```

Comment:
We could maybe move this into it's own issue

Reply:
Do you mean the idea of renaming it to `without_program` like suggested in #48? Or something else?

Reply:
nah the changes to the doc comments as they are pretty much unrelated to the changes here